server/base/src/main/java/org/apache/accumulo/server/util/UpgradeUtil.java

```diff
@@ -48,6 +59,14 @@ static class Opts extends ConfigOpts {
         description = "prepare an older version instance for an upgrade to a newer non-bugfix release."
             + " This command should be run using the older version of software after the instance is shut down.")
     boolean prepare = false;
+
+    @Parameter(names = "--check",
+        description = "check that 'accumulo upgrade --prepare' was run on the instance after it was shut down.")
+    boolean check = false;
```

Comment:
This is naming suggesting, feel free to ignore. 

This seems to do more than check if the 2.1. step was run, it also makes changes that allow the 4.0 upgrade to run.  I think of check as passive and looking at they state of things and not changing anything.  Could possibly use another name that implies something more active is happening.

```suggestion
    @Parameter(names = "--initiate",
        description = "check that 'accumulo upgrade --prepare' was run on the instance after it was shut down and sets persistent markers that allow the manager to run upgrade steps when it next starts.")
    boolean check = false;
```

Reply:
Yeah, I agree that the name is not descriptive of what it's doing. I was thinking of `--make-ready`, but figured that would be considered to verbose.

Reply:
@keith-turner - what about `prevalidate` ?

Reply:
I think `--start`.

My line of thinking is:

I think of `upgrade --prepare` as a currently optional, but in future mandatory, "stage 0". `upgrade --start` would be "stage 1", and starting the manager "stage 2" to complete the upgrade. In Maven, the stage in the release plugin after "prepare" is "perform", but that implies completion, rather than the next stage, and you would still have to start the manager to complete the upgrade. So, that's the basis for my suggesting `--start`. `--begin` is also acceptable, but sounds too formal, and `--init` I think is not quite right (too robotic sounding, and also technically not true, since `--prepare` is kind of an initialize phase).

Reply:
renamed 'check' to 'start' in 376bd50